### PR TITLE
conformance: use `scratch` for minimal test and unblock CI

### PR DESCRIPTION
**Pull request by flouthoc**
Following test is just copying files it should not depend upon `ubuntu:18.04` lets change that so CI never gets blocked.

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by flouthoc** [comment]
@containers/buildah-maintainers PTAL we are mostly using pre-pulled alpine everywhere except this one and the actual test should even work with `scratch`.

**Review by vrothberg — APPROVED** [review]
LGTM

**Comment by nalind** [comment]
/hold
Opinions on leaving this one be to catch possible weirdness like #4551?

**Comment by nalind** [comment]
Well, darn.  Timing really is everything, sometimes.

**Pull request merged**
